server/plugins/org.eclipse.glsp.ide.editor/src/org/eclipse/glsp/ide/editor/utils/UIUtil.java

```diff
@@ -63,13 +62,7 @@ public static void asyncExec(final Runnable runnable) {
       }
    }
 
-   public static Optional<Shell> findShell(final String clientId) {
-      return GLSPIdeEditorPlugin.getDefault().getGLSPEditorRegistry().getGLSPEditor(clientId)
-         .map(editor -> editor.getEditorSite().getShell())
-         .or(UIUtil::findShell);
-   }
-
-   private static Optional<Shell> findShell() {
+   public static Optional<Shell> findShell() {
```

Comment:
Is there a reason why we remove `findShell(clientId)`? Wouldn't it be just as good to keep it and pass the `clientId` as before?

Reply:
I removed it as I did not really see the reason for it. And there is no EditorSite anymore. I added it back by getting the Shell from the Browser Composite